ui/src/components/Orchestration/CreateAccountButton.tsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+
+const CreateAccountButton = ({ handleCreateAccount, loadingCreateAccount }) => (
+  <button
+    className={`daisyui-btn daisyui-btn-info mt-4 w-full self-center text-sm ${loadingCreateAccount ? 'bg-gray-600' : 'bg-blue-500'}`}
+    onClick={handleCreateAccount}
+    disabled={loadingCreateAccount}
+  >
+    {loadingCreateAccount ? (
+      <svg aria-hidden="true" role="status" className="inline w-4 h-4 me-3 text-gray-200 animate-spin dark:text-gray-600" viewBox="0 0 100 101" fill="none" xmlns="http://www.w3.org/2000/svg">
+        <path d="M100 50.5908C100 78.2051 77.6142 100.591 50 100.591C22.3858 100.591 0 78.2051 0 50.5908C0 22.9766 22.3858 0.59082 50 0.59082C77.6142 0.59082 100 22.9766 100 50.5908ZM9.08144 50.5908C9.08144 73.1895 27.4013 91.5094 50 91.5094C72.5987 91.5094 90.9186 73.1895 90.9186 50.5908C90.9186 27.9921 72.5987 9.67226 50 9.67226C27.4013 9.67226 9.08144 27.9921 9.08144 50.5908Z" fill="currentColor"/>
```

Comment:
where do these path numbers come from? how do we maintain this?

Reply:
this is just a raw svg path for a common spinner. I was going to sub it out for a lib component, prob from daisy since its a dep

Reply:
It's often OK to copy from an open source package, but we need to acknowledge the source. And we need to check that the license isn't copyleft; it has to be compatible with ours (Apache 2).